interfaces/builtin/posix_mq.go

```diff
@@ -295,7 +301,7 @@ func (iface *posixMQInterface) generateSnippet(name, plugOrSlot string, permissi
 
 	snippet.WriteString(fmt.Sprintf("  # POSIX Message Queue %s: %s\n", plugOrSlot, name))
 	for _, path := range paths {
-		snippet.WriteString(fmt.Sprintf("  mqueue (%s) \"%s\",\n", aaPerms, path))
+		snippet.WriteString(fmt.Sprintf("  mqueue (%s) type=posix \"%s\",\n", aaPerms, path))
```

Comment:
Do we need to probe for `type=posix` support in the parser? I haven't seen this before so I just want to make sure that any apparmor_parser which supports `mqueue,` also supports `mqueue type=posix,`.

Reply:
Understood. I think we are in the green since we filter for it. I'll make the change.

Reply:
@alexmurray done